### PR TITLE
Custom Reports Update. Fixes #175.

### LINKED ISSUE #175: Custom Reports pulling Curricula w/ DF
[Custom Reports](https://github.com/Capping-CPCA/Web-App/blob/develop/views/reports/custom_reports.php) pulls all curriculum from `curricula` but doesn't check if `curricula.df IS FALSE` .

### PR DESCRIPTION
**Pull request by stickzman**
Fixed deleted curricula being displayed. Changed min range for date to be Jan 1, 2016

Added sex option in custom reports and changed 'Select All' button size to small

Added Locations to custom reports and updated custom table filter display

Added Deselect function to all Select All btns

Fixes #175

### CODE REVIEW

File: views/reports/custom_reports.php [hunk 1]
```diff
@@ -19,11 +19,17 @@
 	
 	global $db;
 
-	$query = "SELECT DISTINCT(curriculumname), curriculumid FROM curricula;";
+	$query = "SELECT DISTINCT(curriculumname), curriculumid FROM curricula WHERE df <> 1;";
```
Comment (ghost) [inline views/reports/custom_reports.php]: While the df flag used to be 0 or 1 it's been updated to be `BOOLEAN` [here](https://github.com/Capping-CPCA/Database/blob/4cce0d1b5169677315c3a2417c32febcfccea3a1/TABLES.sql#L228).

Comment (Scott123180) [reply views/reports/custom_reports.php]: So 10.11.12.33 has the new DB and should be hosting the latest version of the database. The correct way to check should be `WHERE df IS FALSE`

Here's a snippet from the postgres documentation:
image

Comment (stickzman) [reply views/reports/custom_reports.php]: Okay cool, that makes way more sense. Updating now.

**Review by hoenn — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Approved on my main account, sorry 😅

**Review by blackjack26 — APPROVED** [review]

**Pull request merged**